corehq/apps/users/static/users/js/mobile_workers.js

```diff
@@ -112,6 +113,30 @@ hqDefine("users/js/mobile_workers",[
             });
         });
 
+        self.sendConfirmationEmail = function () {
+            var urlName = 'send_confirmation_email';
+            var $modal = $('#confirm_' + self.user_id());
+
+            $modal.find(".btn").addSpinnerToButton();
```

Comment:
Will this also disable the button?  Can the user spam click and send a bunch of emails?  There's a `disable-on-submit` css class that does something along those lines, though you might have to be in a form.

Reply:
you can send multiple emails if you spam click. will look into fixing in a follow up